src/java/org/apache/cassandra/metrics/ClientMetrics.java

```diff
@@ -145,7 +146,15 @@ private int countConnectedClients()
 
     private <T> Gauge<T> registerGauge(String name, Gauge<T> gauge)
     {
-        return Metrics.register(factory.createMetricName(name), gauge);
+        return registerGauge(name, null, gauge);
+    }
+
+    private <T> Gauge<T> registerGauge(String name, String deprecated, Gauge<T> gauge)
```

Comment:
nit: I'd like to avoid the "deprecated" terminology in the actual code (as opposed to in comments, where we can give hints about why aliases exist) in favor of "alias" in as many places as possible. (i.e. We have aliases, and they may exist for a number of reasons, including deprecation?) I'll expand on this in another comment in `TableMetrics`...

Reply:
yes - i wasn't a fan of "deprecated" either.......i wasn't sure how much tolerance folks had though for more drastic changes here. i expected that could be discussed during review. i don't quite remember what else i had in mind though.

Reply:
This one I think we can just change to `alias`. The stuff in `TableMetrics` (see comments there) is a more involved, but not that much more.